doc/dynamo0p3.rst

```diff
@@ -1129,7 +1187,7 @@ inner_product
 
 Computes the inner product of the fields *field1* and *field2*, *i.e.*: ::
 
-  sumval = SUM(field1(:)*field2(:))
+   sumval = SUM(field1(:)*field2(:))
```

Comment:
I wouldn't bother to change if it works OK. I was just wondering if there was a standard that I did not know about.